vividus/src/main/java/org/vividus/configuration/AbstractPropertiesProcessor.java

```diff
@@ -56,8 +54,7 @@ public String processProperty(String propertyName, String propertyValue)
         {
             String partOfValueToProcess = matcher.group(1);
             String processedPartOfValue = processValue(propertyName, partOfValueToProcess);
-            processedValue = processedValue.replace(processorMarker + "(" + partOfValueToProcess + ")",
-                    processedPartOfValue);
+            processedValue = processedValue.replaceFirst(propertyPattern.pattern(), processedPartOfValue);
```

Comment:
why first but not all?

Reply:
because in `propertyPattern.pattern()` inside brackets is not specified particular value. 
as result - with `replaceAll()` will be replaced  not only needed expressions, which were processed.